plugins/jobtype/src/azkaban/jobtype/HadoopSecureSparkWrapper.java

```diff
@@ -0,0 +1,157 @@
+/*
```

Comment:
something like https://github.com/azkaban/azkaban-plugins/blob/master/plugins/jobtype/jobtypes/java/private.properties which specifies which class to use for which jobtype